### ISSUE TITLE
Support for more file extensions

### ISSUE THREAD

**Issue by geekmidas** [serverless-stack/serverless-stack#531]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@serverless-stack/resources@0.32.0` for the project I'm working on.

I am using React in my lambda function and needed my file to use `.tsx` and I realised it wasn't supported by the nodeBuilder

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@serverless-stack/resources/dist/util/nodeBuilder.js b/node_modules/@serverless-stack/resources/dist/util/nodeBuilder.js
index ced3615..8abeeb1 100644
--- a/node_modules/@serverless-stack/resources/dist/util/nodeBuilder.js
+++ b/node_modules/@serverless-stack/resources/dist/util/nodeBuilder.js
@@ -100,9 +100,13 @@ function builder(builderProps) {
     const tsconfig = path.join(srcPath, "tsconfig.json");
     const hasTsconfig = fs.existsSync(tsconfig);
     // Check entry path exists
-    let entryPath = path.join(srcPath, builder_1.addExtensionToHandler(handler, ".ts"));
-    if (!fs.existsSync(entryPath)) {
-        entryPath = path.join(srcPath, builder_1.addExtensionToHandler(handler, ".js"));
+    const exts = ['.ts', '.tsx', '.js', '.jsx'];
+    let entryPath = ''
+    for(let ext of exts) {
+        entryPath = path.join(srcPath, builder_1.addExtensionToHandler(handler, ext));
+        if(fs.existsSync(entryPath)){
+            break;
+        }
     }
     if (!fs.existsSync(entryPath)) {
         throw new Error(`Cannot find a handler file at ${entryPath}".`);
diff --git a/node_modules/@serverless-stack/resources/src/util/nodeBuilder.ts b/node_modules/@serverless-stack/resources/src/util/nodeBuilder.ts
index ec42838..8dc6fab 100644
--- a/node_modules/@serverless-stack/resources/src/util/nodeBuilder.ts
+++ b/node_modules/@serverless-stack/resources/src/util/nodeBuilder.ts
@@ -124,9 +124,13 @@ export function builder(builderProps: BuilderProps): BuilderOutput {
   const hasTsconfig = fs.existsSync(tsconfig);
 
   // Check entry path exists
-  let entryPath = path.join(srcPath, addExtensionToHandler(handler, ".ts"));
-  if (!fs.existsSync(entryPath)) {
-    entryPath = path.join(srcPath, addExtensionToHandler(handler, ".js"));
+  let entryPath = ''
+  const extensions = ['.ts', '.tsx', '.js', '.jsx'];
+  for(let ext of extensions) {
+    entryPath = path.join(srcPath, addExtensionToHandler(handler, ext));
+    if (fs.existsSync(entryPath)) {
+      break;
+    }
   }
   
   if (!fs.existsSync(entryPath)) {
```
I would like to make this fix if no one else is working on it.
<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by fwang** [comment]
Hey @geekmidas, the patch looks good. Can you open a PR with the change?

Also, it'd be great if you can attach an example of a `.jsx` or `.tsx` handler. I'd like to add it to the tests.

**Comment by geekmidas** [comment]
@fwang Where should I add the example? And what is the standard on branch naming when working on a feature?

**Comment by fwang** [comment]
You can just paste it here, or link to a gist. Really what is convenient for you.

We don't have a strict naming currently, but I often prefix the branch with either `bug-` or `feature-`.

**Comment by geekmidas** [comment]
@fwang 
```ts
import React from 'react';
import ReactDOMServer from 'react-dom/server';

function User(props: Event) {
   return (
    <>
        name: {props.name)
    </>
  )
}
export async function handler(event: Event){
  const  { name } = event;
  const text = ReactDOMServer.renderToStaticMarkup(<User name={name} />);

  return {
    text
  }
}
```

**Comment by geekmidas** [comment]
@fwang I am also not allowed to push code in order to make a PR.

**Comment by fwang** [comment]
@geekmidas Released v0.33.0.
